### DISCUSSION TITLE
2024 Q2 - Quarterly Brigade Planning Discussion

### DISCUSSION THREAD

**Discussion by tyliec** [CodeWithAloha/docs#9] [Q&A]
# Overview

Hello, Friends 🖌️ . Once again our [upcoming quarterly brigade planning meeting](https://www.meetup.com/code-with-aloha/events/298029521/) is arriving. This discussion post serves an opportunity to open the floor to any potential feedback and improvements into the group's operations and direction, and is meant to serve as a platform of things to talk about, and is open for anyone to contribute/suggest any comments/concerns/improvements. 🚀 

# Topics

### Reducing Meeting Schedules

Currently, we have 3 regular meetings scheduled (first 3 Mondays of the month). This proposal scales back the meetings that we have, cutting one of the regular monthly meetings to two:
- The monthly social/onboarding events (First Monday of the month)
- A project "workday" (Third Monday of the month)

The end goal for the meetings is to get people engaged so more project work can get done, but we aren't seeing more people get engaged (regularly attend the meetings), so the work output takes a hit since a few of us are more focused on dealing with scheduling/organizing meetings rather than just doing work.

This would scale back our monthly meetings from 3 to 2, with them being spaced out by a week (more recovery time 😆 ).

Personally, I've also scaled back the UIPA.org project workdays to once every two weeks for the same reasons.

### Remote/External Volunteers

So far, we've mostly been taking in and onboarding volunteers residing in Hawaii. The intake of these volunteers has typically been through word of mouth, and the attending of newcomers through [some of our Meetup events](https://www.meetup.com/code-with-aloha/). Roughly speaking, around 5-10 new faces a month (with a much more questionable retention rate).

From discussions with a few of the other brigades in [ACT](https://civictechnologists.org/), there has been some success with taking in external/remote volunteers. To experiment with this, I have previously set up [a profile on DemocracyLab](https://www.democracylab.org/groups/112) and put a few of the group's projects on the platform. This has resulted in a few people reaching out every month since in offer to volunteer with some of the projects.

This does add an additional management burden, because product management will have to occur remotely over multiple time zones.

### Volunteer Oriented Outings & Rewards

In the past few years, we've gone on [a few group field trips](https://github.com/orgs/CodeWithAloha/discussions/3#:~:text=Went%20on%20a%20few%20group%20field%20trips), with the most recent one being [a trip to a honey tasting included bee farm tour](https://www.meetup.com/code-with-aloha/events/298229547).

Traditionally, these outings have been open to all who wish to participate - extremely consistent members and first timers alike.

One of the many pictures:

<img src="https://github.com/CodeWithAloha/docs/assets/15609358/52df3ae1-790a-4760-add6-5482ef33b899" alt="drawing" width="250"/>

The purpose of these outings has been to reward existing volunteers, attract new ones, and increase team camaraderie. Due to the "reward" being spread to all these different groups, it has resulted in everyone getting a little slice. A proposal here would be to double down on rewarding the more consistent volunteers, in particular those who help host meetings. These might occur less frequently, but with more investment towards the reward.

For the less consistent volunteers, we will still be enticing them in the following ways:
- Shouted out on the website via the [All Contributors](https://allcontributors.org/) spec
- An eventual [Holopin](https://holopin.io/) adoption - fun little digital badges (but actually, cool!)

### Fundraising with Swag 😎 

Now that many of our projects have some sort of branding, we can use that as a way to both raise funds and increase awareness of our initiatives.

We've already done some experiments with project related merch, for example the latest experiment:

<img src="https://github.com/CodeWithAloha/docs/assets/15609358/f3a5da2e-4fb6-4e50-a63d-d8dad2a681e9" alt="drawing" width="250"/>

Not only does this help us raise some funds to keep our projects rolling and fund group operations, but it also spreads awareness for our projects in a very cool way 😎 . 

There are sites that provide this as a white label service (i.e. https://printful.com/ or https://printify.com/) - perhaps we can leverage one of these platforms and have an easy way to distribute some product merch :)

### Current Projects

Surveying the existing landscape, our current projects consist of:

- [Hawaii Zoning Atlas](https://hawaiizoningatlas.com/)
- [HIERR](https://github.com/CodeWithAloha/HIERR)
- [UIPA](https://github.com/CodeWithAloha/uipa)
- [Website](https://github.com/CodeWithAloha/website)

We should talk about how each of these are going, and any existing work items that need to be done.

### Our Wins

- Monthly onboarding events are going spectacular :)
- ...

# Fun Animal GIF

![pato-duck](https://github.com/CodeWithAloha/docs/assets/15609358/f5717b13-7e6e-4c44-8769-a1f4c98b4b3b)

# Related Items

- Meetup Link to the upcoming Quarterly Brigade Planning Meeting: https://www.meetup.com/code-with-aloha/events/298029521/
- Previous Q1 Discussion Post: https://github.com/orgs/CodeWithAloha/discussions/5

**Comment by k7lim** [comment]
**Get series on people's calendar**
Meetup is a discovery mechanism but is terrible for establishing the habits/series.
Like unless I'm actively visiting [meetup.com](http://meetup.com/) regularly I won't get CwA on my calendar
We're already on a predictable cadence, and there's gotta be a way to transition from "found your instance on meetup" to "subscribed to your series on __Platform TBD__"

**Reply by tyliec** [reply]
Did some looking, and there is a set of links at the bottom of this page: https://www.meetup.com/code-with-aloha/events/calendar/ to subscribe to an iCal/RSS feed. At least the feature is supported by Meetup, but it definitely isn't easy to find.

**Comment by k7lim** [comment]
**Reduce Onboarding manual steps: make Slack community joining self serve, reinvest saved organizer energy into implementing initial engagement steps**
proof-of-concept: i made a self-service link (and easy URL redirect) for [hawaii.slack.com](http://hawaii.slack.com/) such that mentors/mentees who want on slack can jump on easily : [hawaiitechmentors.org/slack](http://hawaiitechmentors.org/slack)
On CwA Slack, i can make one with my current permissions, but it looks like anything i create will be 30-days expired. i think higher permissions could make an evergreen one, such that it can be QR code or short-linked
So with less time spent on onboarding people into Slack, we could invest some energy in implementing first interactions. A lot of good discords do this, getting in the door is trivial, but once in, they have you use reactions to choose a role, have you ratify terms of community, are prompted to intro yourself, etc. Maybe automatically pairing someone with a project based on a simple questionnaire. Or using "reactions" to declare their own interest and having the PM of that program get them into the GitHub

**Reply by tyliec** [reply]
I've set up a DNS record under `codewithaloha.org` for `http://slack.codewithaloha.org/`, which should redirect to a self-serve Slack invitation link :) Just made it, so it might take a bit for the DNS propagation to take place. After a day or so, can someone test it out and confirm that it works?

**Reply by k7lim** [reply]
slack.codewithaloha.org doesn't resolve for me

**Reply by k7lim** [reply]
still doesn't

**Reply by tyliec** [reply]
It worked for me today - mind trying again? Maybe it just took a bit

**Reply by k7lim** [reply]
Works now on my mobile connection. Will reconfirm on my home fiber, that's the place it wasn't

**Comment by k7lim** [comment]
**Reducing GitHub hurdles, increasing momentum toward first commit**
i don't know if there's an equivalent self-serve github invitation, and probably we don't want it for security purposes. but the next best thing would be like a google doc/form that helps people take the steps to get into the GitHub org, saving hosts/onboarders time. And just like the Slack idea, re-invest some of that time/energy into figuring out their first post-join action

**Comment by k7lim** [comment]
**Follow-up email for people who show potential to be contributors** 
Observing the last meeting, we had a new person show a lot of interest and relevant experience. Was asked by @avenmia to tackle some tasks after the meeting. Haven't heard back just yet, normal, but wouldn't want radio silence to go for too long. 

How might we time an email to their inbox
* Reiterate we're happy they joined
* Encourage them to install the slack app so we can @ them for specific conversations (web Slack tab gets closed, out of sight out of mind)
* Offer to pair them with a buddy?

**Reply by afoolinschool** [reply]
I like the idea of paring folks up with a buddy, as a fairly new member one of the biggest hurdles is just being a bit intimidated by the code base and not really knowing how the sites work currently, how to test, and it would be really helpful to have someone that's assigned to help you since you can feel like you're bugging the same people over and over again. 

Or maybe if not a buddy like an office hour or something, though there's scheduling issues with that I suppose.

**Comment by tyliec** [comment]
Another note from @JerryIsdale:

<img width="999" alt="Screenshot 2024-04-24 at 10 00 21 PM" src="https://github.com/CodeWithAloha/docs/assets/15609358/1d6a8116-f7be-49f1-bea4-86197d50caaa">

**Reply by avenmia** [reply]
Originally the 3rd meeting was a zoom meeting, but we found that since more people came to the in-person meetings, no one showed up to the zoom meeting. If we go down to two meetings, which meeting would be available to remote volunteers?

**Comment by avenmia** [comment]
Discussion about our priorities when taking on a project:
* What types of projects do we want to do?
  * fun game-type projects 
  * Help improve an organization's processes
  * New projects from scratch versus improving an existing one
* How important is open source?
* What types of non-profits would we want to work with?
* What types of government projects would we want?

**Reply by tyliec** [reply]
Attending some flavor to this bullet point and breaking it out of a Slack chat:

The way I've framed the projects we work on to new members to the group are that the projects we work on impact everyone that lives here (essentially meaning that all the projects have some government tie-in, as "government" is the body that we as a society collectively decide to live under). The disconnect that I've had with some past project proposal is that non-profits / private entities conceptually do not have to serve everyone, but in some cases they intuitively do.

Basically in my mind when taking on projects - is the work we are doing creating a lasting and sustainable difference in our community? Or are we just making something shiny for the short term feel good aspect.

And to end my rant, there is no silver bullet here. The projects we take on will almost always be a blend of idealism (long term impact... like multiple generations impact) and the practical (less problems right now, yay!). But I like to think that we try to steer the group towards the former and not the latter 😅  A side effect of which to be honest that is probably resulting in less engagement in this group too lol
